xml/System.Text.Json/JsonException.xml

```diff
@@ -1,4 +1,4 @@
-﻿<Type Name="JsonException" FullName="System.Text.Json.JsonException">
```

Comment:
Not sure why this line remains changed, it's probably the BOM character. I don't think it will matter.